src/bot/utils/getUserNotDaily/getUserNotDaily.service.ts

```diff
@@ -198,13 +198,16 @@ export class UserNotDailyService {
         userNotDaily = await Promise.all(
           notDaily.map((user) =>
             this.userRepository
-              .createQueryBuilder()
+              .createQueryBuilder('user')
               .where(`LOWER("email") = :email`, {
                 email: user.email.toLowerCase(),
               })
               .orWhere(`LOWER("username") = :username`, {
                 username: user.email.toLowerCase(),
               })
+              .andWhere('("createdAt" < :today)', {
+                today: Date.now() - 86400 * 1000,
```

Comment:
why do we need to minus 86400 * 1000? what is it? Please declare a meaningful variable for some magic number.